### ISSUE TITLE
`OCA\GroupFolders\Command\ListCommand::execute()` must be of the type `int`, `null` returned

### ISSUE THREAD

**Issue by francoisPE** [nextcloud/groupfolders#1281]
Hello,
When I run groupfolders CLI command I have always same error.
In web interface seems to work !

Example of command

```
 php occ groupfolders:list
 php occ groupfolders:quota 1 10G
```

Error 

```
An unhandled exception has been thrown:
TypeError: Return value of "OCA\GroupFolders\Command\ListCommand::execute()" must be of the type int, "null" returned.                                        in /var/mypath/apps/mail/vendor/symfony/console/Command/Command.php:258
Stack trace:
#0 /var/mypath/core/Command/Base.php(169): Symfony\Component\Console\Command\Command->run()
#1 /var/mypath/apps/mail/vendor/symfony/console/Application.php(971): OC\Core\Command\Base->run(                                       )
#2 /var/mypath/apps/mail/vendor/symfony/console/Application.php(290): Symfony\Component\Console\                                       Application->doRunCommand()
#3 /var/mypath/apps/mail/vendor/symfony/console/Application.php(166): Symfony\Component\Console\                                       Application->doRun()
#4 /var/mypath/lib/private/Console/Application.php(215): Symfony\Component\Console\Application->                                       run()
#5 /var/mypath/console.php(100): OC\Console\Application->run()
#6 /var/mypath/occ(11): require_once('/mypath...')
```

**Comment by francoisPE** [comment]
Any idea ?

**Comment by kowalski7cc** [comment]
It happens to me on `occ groupfolders:scan 3 -vvv`
```
An unhandled exception has been thrown:
TypeError: Return value of "OCA\GroupFolders\Command\Scan::execute()" must be of the type int, "null" returned. in /var/www/html/custom_apps/mail/vendor/symfony/console/Command/Command.php:258
Stack trace:
#0 /var/www/html/core/Command/Base.php(169): Symfony\Component\Console\Command\Command->run()
#1 /var/www/html/custom_apps/mail/vendor/symfony/console/Application.php(971): OC\Core\Command\Base->run()
#2 /var/www/html/custom_apps/mail/vendor/symfony/console/Application.php(290): Symfony\Component\Console\Application->doRunCommand()
#3 /var/www/html/custom_apps/mail/vendor/symfony/console/Application.php(166): Symfony\Component\Console\Application->doRun()
#4 /var/www/html/lib/private/Console/Application.php(215): Symfony\Component\Console\Application->run()
#5 /var/www/html/console.php(100): OC\Console\Application->run()
#6 /var/www/html/occ(11): require_once('/var/www/html/c...')
#7 {main}
```

**Comment by francoisPE** [comment]
Hello,
I confirm today :

php occ groupfolders:create Sharedfolder
gives
"
An unhandled exception has been thrown:
TypeError: Return value of "OCA\GroupFolders\Command\Create::execute()" must be of the type int, "null" returned. in /mypath/apps/mail/vendor/symfony/console/Command/Command.php:258
"
and 
php occ groupfolders:quota 1 100G
gives
"
An unhandled exception has been thrown:
TypeError: Return value of "OCA\GroupFolders\Command\Quota::execute()" must be of the type int, "null" returned. in /mypath/apps/mail/vendor/symfony/console/Command/Command.php:258

Versions
       "groupfolders": {
            "enabled": "yes",
            "installed_version": "8.2.0",
            "types": "filesystem,dav"
        },
        "groupquota": {
            "enabled": "yes",
            "installed_version": "0.1.4",
            "types": "filesystem"

Many thanks for your help in building a workaround

**Comment by vasyugan** [comment]
If more confirmations are needed: I see the same, with two Nextcloud instances using PHP 7.4 on Ubuntu 20.04

**Comment by pierreozoux** [comment]
@vasyugan and @francoisPE cam you specify your Nc and groupfolder version?

**Comment by francoisPE** [comment]
My conf
NC 20.0.7
groupfolders 8.2.0

**Comment by fschrempf** [comment]
This looks like a duplicate of the already fixed #1153. Please reopen or create a new issue if this is wrong. Thanks!
